src/SOS/Strike/hostcoreclr.cpp

```diff
@@ -377,37 +377,33 @@ static HRESULT GetHostRuntime(std::string& coreClrPath, std::string& hostRuntime
 
         // First attempt find the highest LTS version. We want to start with the LTSs
         // and only use the higher versions if it isn't installed.
```

Comment:
Comment doesn't match code.  Is it deliberate we are not looking for LTS versions first in the code below?

Reply:
I need to change the comment.  I want to search for 5.0 first now that it has shipped (which I realized isn't a LTS now) unless anybody has an objection?